test/include.test.js

```
@@ -690,3 +693,96 @@ function clearAndCreate(model, data, callback) {
     itemIndex++;
   }
 }
+
+describe('Model instance with included relation .toJSON()', function() {
+  var db, ChallengerModel, GameParticipationModel, ResultModel;
+
+  before(function(done) {
+    db = new DataSource({connector: 'memory'});
+    ChallengerModel = db.createModel('Challenger',
+      {
+        name: String
+      },
+      {
+        relations: {
+          gameParticipations: {
+            type: 'hasMany',
+            model: 'GameParticipation',
+            foreignKey: ''
+          }
+        }
+      }
+    );
+    GameParticipationModel = db.createModel('GameParticipation',
+      {
+        date: Date
+      },
+      {
+        relations: {
+          challenger: {
+            type: 'belongsTo',
+            model: 'Challenger',
+            foreignKey: ''
+          },
+          results: {
+            type: 'hasMany',
+            model: 'Result',
+            foreignKey: ''
+          }
+        }
+      }
+    );
+    ResultModel = db.createModel('Result', {
+      points: Number,
+    }, {
+      relations: {
+        gameParticipation: {
+          type: 'belongsTo',
+          model: 'GameParticipation',
+          foreignKey: ''
+        }
+      }
+    });
+
+    async.waterfall([
+      createChallengers,
+      createGameParticipations,
+      createResults,
+      function() {
+        done();
```

ignoring errors here on purpose?